### ISSUE TITLE
Incorrect honor reward

### ISSUE THREAD

**Issue by holzru** [Codewars/codewars.com#243]
Completed a level 3 kata earlier today and only got one honor. This has been happening on and off and correcting itself shortly after. However, a day or two ago I completed a level 4 and only got one honor and it never corrected itself, and today the one honor I got from the level 3 has not corrected iteself. Am I missing something about the honor per kata system or is codewars bugging?
I don't remember which level 4 it was that I was shorted on, but the level 3 that i did today was the Sudoku Solver for python.

**Comment by tselishev-semen** [comment]
I has simular problem. I solve two katas (javascript):
1) http://www.codewars.com/kata/513e7e1aee7d36073e00000d
2) http://www.codewars.com/kata/513e7e1aee7d36073e00000d
And i got 0 honor for each kata. What's I did wrong?

**Comment by jhoffner** [comment]
Honor does not always get immediately rewarded, as its a lower priority background task which might get  queued. If for some reason that task fails (which it shouldn't) then your honor should still eventually update when its periodically recalculated. 

You can debug your points using www.codewars.com/users/{username}/site-events. Please let me know if anything looks off there.

**Comment by holzru** [comment]
Yeah thank you, in the time since I posted it came. I was used to a delay
of a day or two and then all of the sudden a couple of them took more than
a week and a half and hadn't been updated so I thought I'd throw up a red
flag. Thank you for getting back to me.
Russell Holz

**Comment by Sinelife** [comment]
More than one month passed from time when I completed 4kyu Kata "Large Factorials". I get 0 points for it and it didn't marked with tick. I tried few more times to send code in this Kata but result didn't change.  I register other aacount and completed this Kata from it. On other account I get 32 points from "Large Factorials" Kata. 
So what should I do to get honor for this Kata?
Same problem I sometimes met with some other 6kyu Katas.
